### DISCUSSION TITLE
Error reading LoginForm.CSSLibrary: Property CSSLibrary does not exist

### DISCUSSION THREAD

**Discussion by lexidev** [500Foods/TMS-WEB-Core-TemplateDemo#2] [Q&A] [answered]
Hi!
I try to compile this program but I get the following error:

Error reading LoginForm.CSSLibrary: Property CSSLibrary does not exist...

Where is the problem?

WEB Core 2.0.5.0

**Comment by 500Foods** [comment, marked as answer]
I'm not 100% sure, but it sounds like your TMS WEB Coire installation might not be working quite right.  Maybe try reinstalling it.  But be sure to remove it from Windows first using "Add or remove programs"  to be sure the previous install is completely cleared out.

There isn't anything here (to my knowledge) that wouldn't work on any recent TMS WEB Core release, at least > 2.0.

Also, I've moved this to the Issue queue.  Please be sure to report back if you still have trouble or also if you get it working.

**Reply by lexidev** [reply]
I reinstalled it.

I use Delphi 10.4 CE.

Here is the error details (UnitLogin):

![err1](https://user-images.githubusercontent.com/28691915/223505519-58bf5f2b-cf1c-4b56-9e65-46d5c9601317.png)


![ERR2](https://user-images.githubusercontent.com/28691915/223506302-16dd6997-ea9d-4e14-b45e-9a97cce122b4.png)

**Comment by 500Foods** [comment]
Odd.  CSSLibrary is a standard property of TWebForm I believe.  Are you able to create a new project using the TMS Web Bootstrap Application template?  This is what I used when creating the project.  

![image](https://user-images.githubusercontent.com/41052272/223587210-941c8273-ceaf-4693-b4d5-1af683cccf4e.png)

**Reply by lexidev** [reply]
Sorry, everything works, just need to install the software and run Delphi with local administrator rights.

Can we wait part 3 with a desktop/mobile client? ;)

**Comment by 500Foods** [comment]
Glad to hear you got it working!

Making a desktop client would be very easy. Just create a new TMS WEB Core Miletus project and copy over the rest of the source code.  Might take a couple of hours to sort out a few small things (like localStorage issues mainly) but a desktop client would be quite possible.

I thought just the other day, briefly, about making a VCL client. Then I remembered how much fun TMS WEB Core is and forgot all about it 😆 

For mobile clients, well it is a website, so we're most of the way there already. I hope one day that TMS makes Miletus work on iOS and Android. That would make me very happy 😸
